sdk/opa_test.go

```diff
@@ -293,6 +356,79 @@ func TestPartial(t *testing.T) {
 
 }
 
+func TestPartialWithStrictBuiltinErrors(t *testing.T) {
+
+	ctx := context.Background()
+
+	server := sdktest.MustNewServer(
+		sdktest.MockBundle("/bundles/bundle.tar.gz", map[string]string{
+			"main.rego": `
+				package example
+
+erroring_function(number) = output {
+	output := number / 0
+}
```

Comment:
🤔 Interesting choice, btw. I would have gone with an eval-time type error, like `concat(0)`, or something. But this is fine, of course.x

Reply:
Anders and I had a bug in the 🎄 demo (https://github.com/charlieegan3/policing-christmas-trees/pull/37/files#diff-0639d735d787afa283a6562b862fdd3a9b545090203f1e122db0937b7ac4bedfR33) and so had div by zero on the brain!

Happy to change if you think there's a better example.